pkg/rulesConfig/defaultRules/10-ensure-sca-scanner.go

```diff
@@ -29,7 +39,7 @@ func EnsureScaScanner(githubData map[string]*githubConnector.GithubOwner, gitlab
 	return schemaErrors, nil
 }
 
-func githubErrorsRule10(githubData map[string]*githubConnector.GithubOwner) ([]*SchemaError, error) {
+func githubErrorsRule10(githubData map[string]*githubConnector.GithubOwner, sharedRegexExpressions []string) ([]*SchemaError, error) {
```

Comment:
`sharedRegexExpression` is not fully clear on where it belongs. Maybe change it to something with `shell` or something similar, that indicates where this is supposed to appear? Before, it was `run`, which was in the `run` clause.

Reply:
I mean, it makes sense that the variable sent to here has the name `shared` (line 16), but here in the function itself it would be better to give a more specific definition